plugins/obs-qsv11/device_directx9.cpp

```diff
@@ -435,4 +425,4 @@ mfxStatus CD3D9Device::CreateVideoProcessors()
 }
 */
 
-#endif // #if defined(WIN32) || defined(WIN64)
\ No newline at end of file
+#endif // #if defined(WIN32) || defined(WIN64)
```

Comment:
This change is unrelated.

Reply:
I've removed the addition of the `\n` a the end of this file.